cfme/tests/configure/test_help_menu.py

```diff
@@ -0,0 +1,11 @@
+import fauxfactory
+from cfme.utils.appliance.implementations.ui import navigate_to
```

Comment:
**REQUIRED**
Please follow our style guides http://cfme-tests.readthedocs.io/en/master/guides/dev_guide.html#code-style.

> In addition to being broken up into the three sections of standard library, third-party, and the local application, imports should be sorted alphabetically. ‘import’ lines within those sections still come before ‘from ... import’ lines:

Just put a blank line between third-party and the local imports.